cat-launcher/src/components/Sidebar.tsx

```
@@ -0,0 +1,63 @@
+
+  return (
+    <div
+      className={`fixed top-0 left-0 h-full flex flex-col ${
+        isCollapsed ? "w-20" : "w-64"
+      } bg-gray-800 text-white transition-all duration-300`}
+    >
+      <div className="p-4 flex items-center justify-between">
+        {!isCollapsed && (
+          <h1 className="text-2xl font-bold">Cat Launcher</h1>
+        )}
+        <button onClick={toggleSidebar} className="p-2 hover:bg-gray-700 rounded">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)